### PR TITLE
Fixing script location in deploy

### PR DESCRIPTION
**Pull request by kaadhina**
Bug 574832

### CODE REVIEW

**Comment by kaadhina** [comment]
Is the fix already in master?  In progress.
What testing was done? Manual, UTs
Does this fix change the AT/DT contract? No
Does this fix include any database servicing changes? No
Were tests added to cover this to make sure it doesn't regress? User Story 574852

File: Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1 [hunk 1]
```diff
@@ -96,7 +96,7 @@ try
         Write-Verbose "Environment details $environment"
 
         Write-Verbose "Calling Deploy test agent cmdlet"
```
Comment (acesiddhu) [inline Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1]: why didn't E2e RB failing?

Comment (acesiddhu) [inline Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1]: looks fine otherwise

Comment (kaadhina) [inline Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1]: Current E2E runs without update TA. This should be caught by UT, added a story clear test debt for tasks

**Comment by nigurr** [comment]
Approved. Please Add UT for the same in L0 tests

**Comment by vimegh** [comment]
Looks fine.

**Pull request merged**